### PR TITLE
added space info

### PR DESCRIPTION
**Pull request by dickeyxxx**

### CODE REVIEW

File: commands/apps/info.js [hunk 1]
```diff
@@ -10,13 +10,12 @@ let util     = require('util');
...
   function getInfo(app) {
-    let appUrl = `/apps/${app}`;
-    if (context.flags.extended) appUrl += '?extended=true';
     return {
       addons: heroku.apps(app).addons().listByApp(),
-      app: heroku.request({path: appUrl}),
+      app: heroku.request({path: `/apps/${app}`, headers: {'Accept': 'application/vnd.heroku+json; version=3.dogwood'}}),
       dynos: heroku.apps(app).dynos().list().catch(() => []),
       collaborators: heroku.apps(app).collaborators().list().catch(() => []),
+      extended: context.flags.extended ? heroku.request({path: `/apps/${app}?extended=true`}) : {},
```
Comment (ryanbrainard) [inline commands/apps/info.js]: perhaps a TODO to remove this once the variant is merged?

**Comment by ryanbrainard** [comment]
:+1:

**Comment by jmccartie** [comment]
:+1:

File: commands/apps/info.js (hunk 1 above)
Comment (dickeyxxx) [inline commands/apps/info.js]: done

**Pull request merged**